yb-voyager/src/query/queryissue/issues_dml_test.go

```diff
@@ -108,9 +138,38 @@ func testCopyFromWhereIssue(t *testing.T) {
 	conn, err := getConn()
 	assert.NoError(t, err)
 
+	tmpFile, err := os.CreateTemp("/tmp", "copy_where_example.csv")
```

Comment:
I just wrote a `CreateTempFile` in testutils. See if we can use that?

Reply:
Will take a look